app/storage/redis.py

```diff
@@ -27,16 +31,26 @@ def put(self, model, overwrite=True):
             expiry_at = getattr(model, storage_model.expiry_field)
             expires_in = expiry_at - datetime.now(tz=tzutc())
 
-        record_created = self.client.set(
-            name=key_value, value=value, ex=expires_in, nx=not overwrite
-        )
+        try:
+            record_created = self.client.set(
+                name=key_value, value=value, ex=expires_in, nx=not overwrite
+            )
+        except RedisConnectionError:
+            logger.info("retrying redis set command")
```

Comment:
Can we make the log event consistent across the three commands and make the command a property? That way we can easily search for all retries regardless of command, and still see the command that failed.

```suggestion
            logger.info("retrying redis command", command="set")
```

Reply:
Yeah sure thing - I guess there's no value in seeing that a certain type of command failed?